### PR TITLE
Remove dead blog link and update mailing list link

### PR DESCRIPTION
**Pull request by JVillella**
Fixes dead links on homepage. Re #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -155,8 +155,8 @@ <h1 class="logo">CUSEC<br/><span>Canadian University Software Engineering Confer
 		<p>Take into consideration that the cost of professional conferences run into the hundreds of dollars, if not thousands. It costs us much more than $60 per person to run CUSEC.  We get the additional money through sponsorship and we don't make a penny off the conference.  We do this because, like you, we are passionate about Software Engineering.</p>
 
 		<h2>How Do I Stay Informed</h2>
-		<p>We try to make that as easy as possible for you.  We offer you two ways you can stay informed about CUSEC.  You can sign-up to our <a href="http://lists.kopanas.com/mailman/listinfo/cusec-newsletter">newsletter</a> or to our <a href="http://blog.cusec.net">blog's</a> <a href="http://feeds.feedburner.com/cusec"><img src="images/feed-icon-12x12.png" alt="Icon for Feed" /> RSS feed</a>.  It is recommended you sign-up to both because they don't always include the same information.</p>
-		
+		<p>We try to make that as easy as possible for you. You can stay informed about CUSEC by signing-up to our <a href="http://eepurl.com/bnJ89">newsletter</a>.</p>
```
Comment (alexcoco) [inline index.html]: Can you remove the hyphen from `signing-up`?

**Comment by alexcoco** [comment]
Thanks for the PR! I'll merge right after that one comment is addressed.

**Comment by JVillella** [comment]
Hyphen removed! 😄

**Comment by alexcoco** [comment]
Thank you!

**Pull request merged**